src/pj_list.h

```diff
@@ -144,6 +144,7 @@ PROJ_HEAD(vandg2, "van der Grinten II")
 PROJ_HEAD(vandg3, "van der Grinten III")
 PROJ_HEAD(vandg4, "van der Grinten IV")
 PROJ_HEAD(vitk1, "Vitkovsky I")
+PROJ_HEAD(velocity, "Kinematic grid shift")
```

Comment:
Perhaps try to uniformize to one of both terms: velocity or kinematic ? And perhaps the projection name should have gridshift in it for consistency with the hgridshift and vgridshift names (hum, but that would be a long name, so perhaps not a good idea)

Reply:
I've tried a bunch of different variations and it is quite hard to have a short and descriptive name. The best I could come up with was `velocity`, since it somewhat hints at what's at play. The description expands on that, although with some different words. The operation could be called `kgridshift` for "*kinematic gridshift*", but that is not a very descriptive name I think. I am open to suggestions :)

Reply:
@kbevers: "deformation" could be a possibility, since the result of the operation is to model what geodetically is called intraplate deformation. This would also lead to the possibility of using `dgridshift`, for *"deformation gridshift",* in a symmetrical naming scheme.

Reply:
I like `deformation` better than `dgridshift`. Partly because I am seriously considering merging `hgridshift` and `vgridshift` (I didn't realize the possibility of seperating horizontal and vertical grids in the same `PJ` at the time). This would then simply be called `gridshift`. The symmetry would then be lost.

I'll think about it about and merge this once I have decided on a name.

Reply:
deformation is certainly descriptive - and merging gridshifts sounds like a laudable rationalization